hypothesis-python/src/hypothesis/strategies/_internal/numbers.py

```diff
@@ -90,7 +90,19 @@ def do_draw(self, data):
                     data.stop_example(discard=probe < self.start)
                 return probe
 
-        return d.integer_range(data, self.start, self.end, center=0)
+        # For bounded integers, make the bounds and near-bounds more likely.
+        forced = None
+        if self.end - self.start > 127:
```

Comment:
Why cannot we do the same for smaller ranges as well? :thinking:

Reply:
We *could*, but if the range is small there's just no need to - we'll generate the bounds and near-bounds often enough without the extra bias, and for small ranges simplicity is important because it can let Hypothesis detect state-space exhaustion.